.github/workflows/LintAndTest.yml

```diff
@@ -0,0 +1,30 @@
+name: Lint & Test
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
```

Comment:
Does the presence of `edited` mean that the linter and tests will run every time you edit a PR description / comment?

Reply:
removed